src/main/java/it/pagopa/wallet/config/RequestFilter.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.wallet.config;
+
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.server.WebFilter;
+import org.springframework.web.server.WebFilterChain;
+import reactor.core.publisher.Mono;
+import reactor.util.context.Context;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@Slf4j
+public class RequestFilter implements WebFilter {
```

Comment:
Why this the usage of this filter?

Reply:
No use for now. It could be useful in further features. But i can remove it.